doc/ROS2/setup/microROS_Native.md

```diff
@@ -28,7 +29,6 @@ colcon build
 source install/local_setup.bash
 ```
 
-
 ## Create static micro-ROS library
```

Comment:
suggest to add "for DroidControlShip" to the heading 

This makes it clearer that we are now  using stuff from this Repo, not microROS.

Also a reasoning sentence whould be helpful here, like :

*The static library is needed as DroidControlShip is build outside of the ROS2 environment with PlatformIO and.*